scripts/nicklist.pl

```diff
@@ -222,19 +264,14 @@ sub screen_size {
 		close(TTY);
 		($row, $col, $xpixel, $ypixel) = unpack('S4', $winsize);
 	}
-	
+
 	# set screen width
 	$irssi_width = $col-$nicklist_width-1;
 	$height = $row-1;
-	
-	# on some recent systems, "screen -X fit; screen -X width -w 50" doesn't work, needs a sleep in between the 2 commands
-	# so we wait a second before setting the width
```

Comment:
Changing this to the simpler method also seems like a downgrade

Reply:
true, it looks odd. but seems to be what the author wanted --> https://github.com/coekie/irssi-scripts/commit/6d127c24b5648c7b51c3e85926c7804e0234ce4d#diff-7753d7712b1505f36887cc9678217810L201